src/utils/pst.ts

```diff
@@ -0,0 +1,80 @@
+import axios from "axios";
+
+// ArDrive Profit Sharing Community Smart Contract
+const vertoCacheURL =
+  "https://v2.cache.verto.exchange/-8A6RexFkpfWwuyVO98wzSFZh0d6VJuI-buTJvlwOJQ";
+
+// Calls the ArDrive Community Smart Contract to pull the fee
```

Comment:
Add some clarity about the nature of the number that's coming back, e.g. "A return value of .15 means that the retrieved tip setting is 15% of the data upload cost."

Reply:
[Updated](https://github.com/th8ta/ArConnect/pull/33/commits/f76faa99ec9bfb48f340cbe9b4857e805e4e2582)